README.md

```diff
@@ -1,8 +1,10 @@
 # Architecture Center Samples
 
-Terraform samples for [Cloud Architecture
-Center](https://cloud.google.com/architecture/)
+This repo contains Terraform samples for [Cloud Architecture
```

Comment:
I think an update to 

This repo contains supporting code artifacts for Cloud Architecture